### PR TITLE
Add in Error handling to when provisioning an instance if it cannot b…

### PR DESCRIPTION
**Pull request by seamuslee001**
…e provisioned

### CODE REVIEW

File: dms_instance.module.orig [hunk 1]
```diff
@@ -0,0 +1,143 @@
+<?php
```
Comment (Edzelopez) [inline dms_instance.module.orig]: Did you mean to commit this in?

**Comment by Edzelopez** [comment]
Apart from the one comment I think this is good.

File: dms_instance.module.orig (hunk 1 above)
Comment (seamuslee001) [reply dms_instance.module.orig]: nope removed now

File: dms_instance.module [hunk 2]
```diff
@@ -51,13 +50,17 @@ function dms_instance_dms_instance_insert(EntityInterface $entity) {
...
+  require_once($module_path . DIRECTORY_SEPARATOR . 'dms_instance_secrets.inc');
   $instanceStatus = $entity->get('instance_status')->value;
   if ($instanceStatus == 16) {
     $uuid = $entity->uuid();
+    /** @var Drupal\dms_instance\Queue\DMSInstanceQueue; $queue */
     $queue = new DMSInstanceQueue($entity->getAegirInstance(), Database::getConnection());
     $queue->createQueue();
     $queue->createItem([
-      'url' => $entity->getName() . '-dms.canadahelps.org',
+      'url' => $entity->getName() . '-dms.' . SITE_DOMAIN_HOST,
```
Comment (Edzelopez) [inline dms_instance.module:63]: Where is SITE_DOMAIN_HOST defined?

Comment (seamuslee001) [reply dms_instance.module:63]: its in the dms_instance_secrets.inc file which is excluded from being stored in this repository

**Pull request merged**